Point3D source_pos
Point3D mic_pos
---
FloatArray rir